src/localization/en/app.json

```diff
@@ -118,8 +119,10 @@
     "CLAIM_REWARDS_NOTIFICATION": "Your claimed rewards will appear in your wallet once your transaction confirms.",
     "CLAIM_REWARDS": "Claim rewards",
     "CLAIM_YOUR_REWARDS_DESCRIPTION": "Withdraw all your claimable staking and trading rewards.",
+    "CLAIM_YOUR_REWARDS_DETAILED_DESCRIPTION": "Total claimable rewards includes both your staking and trading rewards. Claiming will withdraw all available rewards to your wallet.",
```

Comment:
```suggestion
    "CLAIM_YOUR_REWARDS_DETAILED_DESCRIPTION": "Total claimable rewards include both your staking and trading rewards. Claiming will withdraw all available rewards to your wallet.",
```

Reply:
If rewards is the subject, then I think `include` is plural and `includes` for a singular reward

Reply:
Hmmmm, this one's interesting. I think `Total claimable rewards` is the subject and it's singular because of the `Total`.